Readme.md

```diff
@@ -1,32 +1,44 @@
+# trash
```

Comment:
I generaly don't like when the library name is the title of a readme. I don't know why it's just a personal preference.